guacamole-docker/bin/start.sh

```diff
@@ -205,6 +205,32 @@ END
         "mysql-user-required" \
         "$MYSQL_USER_REQUIRED"
 
+    set_optional_property \
+        "mysql-ssl-mode"  \
+        "$MYSQL_SSL_MODE"
+
+    set_optional_property        \
+        "mysql-ssl-trust-store"  \
+        "$MYSQL_SSL_TRUST_STORE"
+
+    # For SSL trust store password, check secrets, first, then standard env variable
+    if [ -n "$MYSQL_SSL_TRUST_PASSWORD_FILE" ]; then
+        set_property "mysql-ssl-trust-password" `cat $MYSQL_SSL_TRUST_PASSWORD_FILE`
```

Comment:
The `$MYSQL_SSL_TRUST_PASSWORD_FILE` variable will need to be quoted in case it expands to a value containing whitespace, as does backtick-quoted command (in case the command outputs a value containing whitespace). For example:

    "`cat "$FOO"`"

Same for the other invocations.